src/components/Card/README.md

```diff
@@ -0,0 +1,308 @@
+# Card
+
+Cards contain content and actions about a single subject.
+
+## Basic Usage
+
+To implement the Card component, you need to import at least Card and
+[CardBody](#cardbody) components:
+
+```js
+import { Card, CardBody } from '@react-ui-org/react-ui';
+```
+
+And use it:
+
+```docoff-react-preview
+<Card>
+  <CardBody>
+    Hello!
+  </CardBody>
+</Card>
+```
+
+See [API](#api) for all available options.
+
+## General Guidelines
+
+- Default card is **designed for non-white background.** You may want to either
+  use the [raised variant](#raised-card) or
+  [customize](/docs/customize/theming/overview) the default appearance to make the
+  card stand out on white surfaces.
+
+- Use optional [CardBody](#cardbody) and [CardFooter](#cardfooter) components to
+  provide your content with expected spacing.
+
+- Use **medium or low-emphasis buttons** when there are more cards, e.g. in a
+  grid. It will help you keep the UI clean and easy to scan.
+
+- **Card, or Paper?** Card is a versatile surface for displaying content.
+  However, there is also the [Paper](/components/Paper) component. While Card
+  is designed for displaying items (and also supports more visual options),
+  Paper is usually used to hold larger content areas like lists, grids, or
+  forms.
+
+## Card Composition
+
+Aside from the [CardBody](#cardbody) element, you can add a
+[CardFooter](#cardfooter) to better separate your card's actions from the rest
+of the content.
+
+Import all necessary components:
+
+```js
+import { Card, CardBody, CardFooter } from '@react-ui-org/react-ui';
+```
+
+And use them:
```

Comment:
The multi component composition is shown here as an example. There is a different format used for `Modal` . I have no preference which one is better, but it should be unified.

Could @adamkudrna please take a look at it?